### DISCUSSION TITLE
the good intention was there!

### DISCUSSION THREAD

**Discussion by hpoperator86** [supermerill/SuperSlicer#3984] [General]
@supermerill 
"from november i will be working full time on superslicer" ......... "last activity 7 days ago".... exactly my humor! 

I'm confident that switching to OrcaSlicer was the better move than always having to wait for things to progress here.
The work you still have to catch up on is beyond the capacity of one person. So many bug fixes that still need to be ironed out. 
And then there's the state of the art to catch up with the other slicers.
I think it's a shame that such promises are made and such confidence is stirred up and then almost nothing happens!

**Comment by DannyKlenz** [comment]
You could not have made it more obvious that you have no idea what you are talking about.

**Comment by dbUniBS** [comment]
i mean, he released a new version last week. And he also opened an issue on PrusaSlicer meaning that he is looking into it for the merge he talked about.

I don't think it's that easy to merge after this many months. Ensuring as little as possible will be screwed up is not an easy job. Just give him time.

**Comment by deadcomic** [comment]
@hpoperator86 with that kind of attitude I don't think anyone will miss you, man. Orcaslicer is a good slicer as well but you will likely come back here when you see that it has the same limitations you came here for in the first place. Wish you the best and we hope that when updates do start coming in that you are able to have this same energy toward the progress and development that is needed for this to take place. 

Hope you get the support you are looking for and desperately need.

**Comment by DaTTcz** [comment]
Yes, PrusaSlicer has made progress in the time that nothing has happened here. OrcaSlicer seized the chance to grab a piece for himself. But not everything works as it should for both of them, and you can see that some features deserve a little more time and testing. It's important that these projects are and value each other. For me, it's not important to release one version after another, but to make sure that what's in the current version works. And I have that feeling here. So please let's give peace of mind and rather test what works and what doesn't work. That will certainly be more beneficial than talking about where the new version is...

**Comment by anonymousaga** [comment]
Look, if you dont appreciate Superslicer, leave it be. No need to dishearten supermill - tons of us appreciate it.

I came from PrusaSlicer specifically for XY-hole compensation. And while I have got bugs after a year of no updates (brims dont work anymore), I still keep coming back here.

Orcaslicer - weird fan control issues. And its the most clunky UI I've seen.
PrusaSlicer - doesn't have the niche settings I use.

So I am glad that he is back working on things, and I don't mind the wait.

Keep it nice. Thanks :)

**Comment by supermerill** [comment]
currently, i'm merging with prusaslicer 2.7
I can't make a commit until it's finished.
And there is 200 files, some more than 8k lines to review...
I'm communicating a bit more on discord.

**Reply by darco415** [reply]
I say  @hpoperator86 buys supermerill a cup of coffee

**Reply by Arthur-de-Partuur** [reply]
I say we all buy @supermerill a(nother) cup of coffee!

**Comment by jhstatewide** [comment]
> I think it's a shame that such promises are made and such confidence is stirred up and then almost nothing happens!

What promises were broken and how much have you donated?

**Comment by confuzled-dev** [comment]
@hpoperator86 this aged like milk
